composer.json

```diff
@@ -9,6 +9,11 @@
         "php": ">=5.3.3"
     },
 
+    "require-dev": {
+        "php-yaoi/php-yaoi": "1.0.18",
```

Comment:
do we actually need to use an exact match ? If the library follows semver, a semver constraint would be better